docs/README.tvOS.md

```diff
@@ -1,7 +1,7 @@
 ![Kodi Logo](resources/banner_slim.png)
 
 # tvOS build guide
-This guide has been tested with macOS 10.13.4(17E199) High Sierra and 10.14.4(18E226) Mojave on Xcode 9.4.1(9F2000) and Xcode 10.2(10E125). It is meant to cross-compile Kodi for tvOS 11+ (AppleTV 4/4K) using **[Kodi's unified depends build system](../tools/depends/README.md)**. Please read it in full before you proceed to familiarize yourself with the build procedure.
+This guide has been tested with macOS 10.15.2 Mojave and Xcode 11.3.1. Please note this is the only version our CI system builds with. There is no assurances that other combinations will work correctly. It is meant to cross-compile Kodi for tvOS 11+ (AppleTV 4/4K) using **[Kodi's unified depends build system](../tools/depends/README.md)**. Please read it in full before you proceed to familiarize yourself with the build procedure.
```

Comment:
intended to have tvOS on a different OS version than macOS and iOS builds?

Reply:
the 3rd CI server that solely builds tvos uses Catalina. The other 2 are using Mojave as documented, so yes, at this time its intended.

Reply:
I suggest to keep Mojave (10.14.6) as I've successfully tested this setup multiple times. Btw 10.15 is Catalina :)

Reply:
The intent is to keep it inline with whatever CI uses. I could say Catalina and Xcode 12, but what happens when I move on and that combination is no longer tested/used. At least with Jenkins it runs constantly and we can guarantee the combo works at any given point

Reply:
Also, was a C&P error, I'll fix it